### PR TITLE
tools/rados: add more options to rados benchmark

### PR DESCRIPTION
**Pull request by ifed01**
This includes objects contents reading and omap retrieval (using both single key mode and prefix bulk mode) and cleanup.

In fact that's another fork from #37496 to separate modifications orthogonal to PG removal optimizations to make the latter simpler for reviewing.

Signed-off-by: Igor Fedotov <igor.fedotov@croit.io>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [x] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests
- Teuthology
  - [ ] Completed teuthology run
  - [ ] No teuthology test necessary (e.g., documentation)

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: doc/man/8/rados.rst [hunk 1]
```diff
@@ -132,6 +132,22 @@ Bench options
...
 
+.. option:: --clear-omap
+
+   Clear omap contents.
+
+.. option:: --read-object
+
+   Read contents from the objects.
+
+.. option:: --write-xattr
```
Comment (rubenk) [inline doc/man/8/rados.rst:143]: should this be --read-omap?

Comment (ifed01) [reply doc/man/8/rados.rst]: Yeah, sure. Fixed. Thanks!

**Comment by neha-ojha** [comment]
jenkins test make check

**Comment by neha-ojha** [comment]
@markhpc could you take a look at this?

**Pull request closed without merging**